packages/fxa-settings/src/components/LocaleToggle/index.test.tsx

```diff
@@ -19,23 +20,37 @@ const mockUseLocaleManager = jest.fn(() => ({
     { code: 'he', name: 'Hebrew', nativeName: 'עברית', rtl: true }
   ],
   switchLocale: mockSwitchLocale,
+  clearLocalePreference: mockClearLocalePreference,
+  isUsingBrowserDefault: false,
   isLoading: false
 }));
 
 jest.mock('../../lib/hooks/useLocaleManager', () => ({
   useLocaleManager: () => mockUseLocaleManager()
 }));
 
+// Mock getBrowserDefaultLocaleInfo
+jest.mock('../../lib/locales', () => ({
+  ...jest.requireActual('../../lib/locales'),
+  getBrowserDefaultLocaleInfo: () => ({
+    code: 'en',
```

Comment:
Could be a separate PR since this touches a lot of files but we reference `'en'` a lot and could use `DEFAULT_LOCALE`.